### ISSUE TITLE
Post-deployment validation

### ISSUE THREAD

**Issue by flaviodsr** [goharbor/harbor#13217]
**Is your feature request related to a problem? Please describe.**
After deploying Harbor I would like to be sure that everything is working as expected at least in regard of the functionalities provided by Harbor.

**Describe the solution you'd like**
Provide a way to easily perform basic functionality testing against an existing Harbor deployment.

**Describe the development plan you've considered**
A possible way to implement that could be to also ship an image including basic functionality tests that are already being executed by the CI (such as the `API_DB` tests).
Basically, as part of the release also build and publish an image similar to https://github.com/goharbor/harbor/blob/master/tests/e2e-image/Dockerfile . With that, document how can someone can use that image to validate its Harbor deployment.

**Additional context**
The mentioned image can also be used on `harbor-helm` to implement [Chart Tests](https://helm.sh/docs/topics/chart_tests/) to perform a post-deployment validation.

**Comment by xaleeks** [comment]
have you looked into the health API? @flaviodsr

**Comment by flaviodsr** [comment]
@xaleeks yes. The health API is also part of `API_DB` tests but I believe it is not enough for a full validation of the deployment.

**Comment by reasonerjt** [comment]
@flaviodsr Thanks for the suggestion, if this issue is tartgeting the chart specifically, could you please open the issue in `harbor-helm` repo?

Overall I think it's extremely helpful if we can run the API test in the CI.

Normally before each release of the chart we run the API tests in the private network, it would offload some work if some of the tests can be integrated via github action.

**Comment by flaviodsr** [comment]
@reasonerjt thanks. I've opened an issue at goharbor/harbor-helm#805
